index.js

```diff
@@ -217,7 +217,11 @@ function createMainWindow (done) {
 
     var INDEX = 'file://' + path.resolve(__dirname, './index.html')
     if (!win) {
-      win = new BrowserWindow({ title: APP_NAME, show: false })
+      win = new BrowserWindow({
+        title: APP_NAME,
+        show: false,
+        icon: path.resolve(__dirname, './static/mapeo_256x256.png')
```

Comment:
This is great, can you instead use `path.join` to build the path, and without slashes, like:

```
path.join(__dirname, 'static', 'mapeo_256x256.png')
```
This will work on all platforms by normalizing the path

Reply:
Hi @karissa, great idea! Fixed at c79781d.